processing-common/src/main/kotlin/com/livefront/sealedenum/internal/common/spec/SealedEnumOrdinalPropertySpec.kt

```diff
@@ -8,7 +8,6 @@ import com.squareup.kotlinpoet.TypeName
 import javax.lang.model.element.TypeElement
 
 internal data class SealedEnumOrdinalPropertySpec(
-    private val sealedClass: SealedClass,
```

Comment:
Was this just unused then?

Reply:
Yep, the update to detekt picked up the unused variable, so I just removed it.